static/tests/whisper.js

```diff
@@ -400,71 +196,91 @@ describe('Whisper Tests', function () {
         });
 
         it('ensure symkey exists', function () {
-            keyId1 = node1.shh.generateSymmetricKey();
-            assert.lengthOf(keyId1, 64);
-            expect(node1.shh.hasSymmetricKey(keyId1)).to.equal(true);
+            keyID1 = node1.shh.newSymKey();
+            assert.lengthOf(keyID1, 64);
+            expect(node1.shh.hasSymKey(keyID1)).to.equal(true);
```

Comment:
This also looks like duplicate. We've already tested it in the first context, why do it again?